Caltech-IPAC/firefly
# DM-15157: Image plotting refactor

- created a framework for processing whole FITS file (fitseval)
 - improved image reading, no more 2D array
 - improved mask performance
 - broke up FITSRead into multiple files
 - created a new package (plotdata) for FitsRead related processing
 - improved cacheing but still needs more work
 - moved header processing the the client
 - recognize multiple WCS from file

_to test:_
There really are no new features. The following should work:
 - load image  from a server
 - upload an image
 - upload a lsst image and add all the mask

https://irsawebdev9.ipac.caltech.edu/dm-15157-image-refactor/firefly/
